lib/home/view/user_search_screen.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+import 'package:fpb/authentication_with_firebase/application/bloc/auth_bloc.dart';
```

Comment:
- The search icon doesn't work on my end, so I'm unable to test the search feature. Is there a particualr reason why?
- The other components on the bottom navbar are invisible until you click on it as seen below.

![pr-rev](https://user-images.githubusercontent.com/17765231/221698378-8addce67-7c81-47ae-905b-81a9eb381dc3.jpeg)